content/en/docs/application-store/built-in-apps/radondb-postgresql-app.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Deploy RadonDB PostgreSQL on KubeSphere"
+keywords: 'KubeSphere, Kubernetes, Installation, RadonDB PostgreSQL'
+description: 'Learn how to deploy RadonDB PostgreSQL from the App Store of KubeSphere and access its service.'
+linkTitle: "Deploy RadonDB PostgreSQL on KubeSphere"
+weight: 14294
+---
+
+[RadonDB PostgreSQL](https://github.com/radondb/radondb-postgresql-kubernetes) is an open source, cloud-native, and highly available cluster solution based on [PostgreSQL](https://postgresql.org) database system.
+
+This tutorial demonstrates how to deploy RadonDB PostgreSQL from the App Store of KubeSphere.
+
+## Prerequisites
+
+- Please make sure you [enable the OpenPitrix system](../../../pluggable-components/app-store/).
+- You need to create a workspace, a project, and a user account (`project-regular`) for this tutorial. The account needs to be a platform regular user and to be invited as the project operator with the `operator` role. In this tutorial, you log in as `project-regular` and work in the project `demo-project` in the workspace `demo-workspace`. For more information, see [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/).
+
+## Hands-on Lab
+
+### Step 1: Deploy RadonDB PostgreSQL from the App Store
+
+1. On the **Overview** page of the project `demo-project`, click **App Store** in the upper-left corner.
+
+2. Click **Database & Cache** under **Categories**.
+
+   ![RadonDB PostgreSQL-in-app-store](/images/docs/appstore/built-in-apps/radondb-postgresql-app/radondb-postgresql-in-app-store.png)
+
+3. Find RadonDB PostgreSQL and click **Deploy** on the **App Information** page.
+
+   ![deploy-RadonDB PostgreSQL](/images/docs/appstore/built-in-apps/radondb-postgresql-app/deploy-radondb-postgresql.png)
+
+4. Set a name and select an app version. Make sure RadonDB PostgreSQL is deployed in `demo-project` and click **Next**.
+
+   ![confirm-deployment](/images/docs/appstore/built-in-apps/radondb-postgresql-app/confirm-deployment.png)
+
+5. In **App Configurations**, you can use the default configuration or customize the configuration by editing the YAML file. When you finish, click **Deploy**.
+
+   ![set-app-configuration](/images/docs/appstore/built-in-apps/radondb-postgresql-app/set-app-configuration.png)
+
+6. Wait until RadonDB PostgreSQL is up and running.
+
+   ![RadonDB PostgreSQL-running](/images/docs/appstore/built-in-apps/radondb-postgresql-app/radondb-postgresql-running.png)
+
+### Step 2: View PostgreSQL Cluster status
+
+1. On the **Overview** page of the project `demo-project`, you can see a list of resource usage in the current project.
+
+   ![project-overview](/images/docs/appstore/built-in-apps/radondb-postgresql-app/project-overview.png)
+
+2. In **Services** under **Application Workloads**, click the Service name of RadonDB PostgreSQL.
```

Comment:
What is the purpose of this step? It looks a little bit confusing to me.

Reply:
delete the line